e2e_tests/tests/task/test_generic_tasks.py

```diff
@@ -0,0 +1,190 @@
+import pathlib
+import subprocess
+
+import pytest
+
+from determined.cli import ntsc
+from determined.common import context, util
+from determined.common.api import bindings
+from tests import api_utils
+from tests import config as conf
+
+
+@pytest.mark.e2e_cpu
+def test_create_generic_task() -> None:
+    """
+    Start a simple task with a context directory called from the task CLI
+    """
+    command = [
+        "det",
+        "task",
+        "create",
+        conf.fixtures_path("generic_task/test_config.yaml"),
+        "--context",
+        conf.fixtures_path("generic_task"),
+    ]
+
+    subprocess.run(command, universal_newlines=True, stdout=subprocess.PIPE, check=True)
+
+
+@pytest.mark.e2e_cpu
+def test_generic_task_config() -> None:
+    """
+    Start a simple task without a context directory and grab its config
+    """
+    test_session = api_utils.determined_test_session()
+
+    with open(conf.fixtures_path("generic_task/test_config.yaml"), "r") as config_file:
+        # Create task
+        config = ntsc.parse_config(config_file, None, [], [])
+        config_text = util.yaml_safe_dump(config)
```

Comment:
I am not sure what's the point of doing an extra read/dump round trip when you are not adding any extra configs. e.g. you can read the file straight in: 
```suggestion
        config_text = config_file.read()
```